tests/test_mini_analyses.py

```diff
@@ -386,14 +390,21 @@ def test_load_corrected_positions(self):
                      "No db access, cannot test!")
     def test_nv_event_display(self):
         """
-        Test NV event display for a time range without data (should fail)
+        Test NV event display for a single event.
         """
-        self.st.make(nt_test_run_id, 'events_nv')
-        ev_nv = self.st.get_array(nt_test_run_id, 'event_positions_nv')
-        self.assertFalse(ev_nv, "this test assumes NV events are empty")
+        events_nv = self.st.get_array(nt_test_run_id, 'events_nv')
+        warning = ("Do 'rm ./strax_test_data' since your *_nv test "
+                   "data in that folder is messing up this test.")
+        self.assertTrue(len(events_nv), warning)

```

Comment:
Nope! This is because the previous URL had the same hash (I'm not going to untar-tar-upload-change etc just to change this since the only two people who will notice are you and me).

So that it's stored is fine, as long as it is fetched with the new test-data.

Reply:
the old test data did not have any length

Reply:
Okay I see. In that case I am fine too.